### PR TITLE
populate repository for GLORIA data set pull

### PR DESCRIPTION
**Pull request by steeleb**
Heya Katie!

Okay, super basic PR here. This is 90% a copy of the architecture from the Landsat_C2_SRST repository. The code has not been changed.

Please review the README file in particular, that has changed and outlines the changes I'll be making to the codebase in the next PR.

I'm intentionally tracking the 'data' folder here, since it contains locs that I want to travel with the repo. 

THANK YOU.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
...
+
+Code to pull, collate, and correct Landsat SR product at GLORIA and other sites as presented in Maciel et al. 2023.
+
+This repository is covered by the MIT use license. We request that all downstream uses of this work be available to the public when possible.
+
+## Code Origin
+
+The code presented in this repository is an adaptation of the code in the [ROSSyndicate repository Landsat_C2_SRST](https://github.com/rossyndicate/Landsat_C2_SRST). This code was written by members of the ROSSyndicate at Colorado State University and other contributors. The alterations from the original workflow are as follows:
+
+1.  only targets that pull buffered location data were retained (i.e. no center points or whole waterbodies)
```
Comment (kathryn-willi) [inline README.md]:
```suggestion
1.  only targets that pull buffered location data are utilized (i.e. no center points or whole waterbodies are pulled)
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,23 @@
...
+
+2.  the buffer method has been altered from a circular buffer to a square buffer to match the methods in Maciel, et al 2023.
+
+3.  all thermal data and related columns have been removed for the purposes of this analysis.
+
+Not all location information to replicate the Maciel, et al. analysis using more stringent filters and correction coefficients. However, the locations of all GLORIA data are listed in the 'data' folder within the file "GLORIA_meta_and_lab.csv" and the filtered dataset from Maciel, et al is present in the "Matchups.csv".
```
Comment (kathryn-willi) [inline README.md]: Not quite sure what you mean in the first sentence - is there a typo? o_O

**Review by kathryn-willi — APPROVED** [review]
Looks great to me! Note the minor tweaks to the readme I suggest. :)

File: README.md (hunk 2 above)
Comment (steeleb) [reply README.md]: lols. I'll fix this. This was a stray thought.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,23 @@
...
+
+The code presented in this repository is an adaptation of the code in the [ROSSyndicate repository Landsat_C2_SRST](https://github.com/rossyndicate/Landsat_C2_SRST). This code was written by members of the ROSSyndicate at Colorado State University and other contributors. The alterations from the original workflow are as follows:
+
+1.  only targets that pull buffered location data are utilized (i.e. no center points or whole waterbodies are pulled)
+
+2.  the buffer method has been altered from a circular buffer to a square buffer to match the methods in Maciel, et al 2023.
+
+3.  all thermal data and related columns have been removed for the purposes of this analysis.
+
+Not all location information to replicate the Maciel, et al. analysis using more stringent filters and correction coefficients. However, the locations of all GLORIA data are listed in the 'data' folder within the file "GLORIA_meta_and_lab.csv" and the filtered dataset from Maciel, et al is present in the "Matchups.csv".
```
Comment (steeleb) [inline README.md]:
```suggestion
Not all location information to replicate the Maciel, et al. analysis using more stringent filters and correction coefficients were available. However, the locations of all GLORIA data are listed in the 'data' folder within the file "GLORIA_meta_and_lab.csv" and the filtered dataset from Maciel, et al is present in the "Matchups.csv". The filtered dataset likely represents a fraction of the sites from the upstream pre-filtered datset.
```

**Comment by steeleb** [comment]
Thank you KW! Merging now!

**Pull request merged**